doc/manipulating_visualizing/data_preparation.rst

```diff
@@ -112,10 +123,12 @@ in memory:
 Controlling how the mask is computed from the data
 -----------------------------------------------------
 
-In the basic tutorial, we showed how the masker could compute a mask
-automatically, and it has done a good job. But, on some datasets, the
-default algorithm performs poorly. This is why it is very important to
-**always look at how your data look like**.
+In this tutorial, we show how the masker object can compute a mask
+automatically for subsequent statistical analysis.
+On some datasets, the default algorithm may however perform poorly.
+This is why it is very important to
+**always look at your data** before and after feature
+engineering using masker objects.
```

Comment:
yes. Feature Agglomeration woould be plain Masking, but Feature Engineering implies something smarter than just selecting.